extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -20,11 +20,17 @@ const TrackIcon = ( { isPlaying, isError, className } ) => {
 
 	if ( isError ) {
 		name = 'error';
-		/* translators: This is text to describe the current state. This will go before the track title, such as "Error: [The title of the track]" */
+		/*
+		 * Translators: This is text to describe the current state. This will go
+		 * before the track title, such as "Error: [The title of the track]".
+		 */
 		hiddenText = __( 'Error:', 'jetpack' );
 	} else if ( isPlaying ) {
 		name = 'playing';
-		/* translators: Text to describe the current state. This will go before the track title, such as "Playing: [The title of the track]" */
+		/*
+		 * Translators: Text to describe the current state. This will go before the
+		 * track title, such as "Playing: [The title of the track]".
+		 */
```

Comment:
Have you tested if this comment annotation gets picked up as expected?

Reply:
Good catch! I didn't know that this is being picked up. I guess that we're using the [`babel-plugin-makepot`](https://github.com/WordPress/gutenberg/tree/3077c6ac54dcf127f8573d241489f4c5a3bbdc9d/packages/babel-plugin-makepot) for this. I've ran some tests, and it looks like the [extracting is case-insensitive](https://github.com/WordPress/gutenberg/blob/3077c6ac54dcf127f8573d241489f4c5a3bbdc9d/packages/babel-plugin-makepot/test/index.js#L76) regarding the `translators:` keyword but it won't pick up multiline comments unless that keyword is in the very first line of the comment. FYI, this would also be not good:

```js
/* translators: first line
 * second line
 * third line
 */
```
...because the extracted text would contain the `*`s that come after a newline.

So, the only accepted format for multiline comments like this one would be:

```js
/* translators: first line
   second line
   third line */
```

Reply:
How about keeping translator comments separate then, and in a single line?

Reply:
Some of them were pretty long, so I ended up wrapping them. Fixed in https://github.com/Automattic/jetpack/pull/15281/commits/116bc2b09cbb167da38709b588dc5e4296ed4b5f.